library/std/src/sys/unix/fs.rs

```diff
@@ -1118,6 +1118,19 @@ impl fmt::Debug for File {
             Some(PathBuf::from(OsString::from_vec(buf)))
         }
 
+        #[cfg(target_os = "freebsd")]
+        fn get_path(fd: c_int) -> Option<PathBuf> {
+            const KINFO_FILE_SIZE: libc::c_int = 1392;
```

Comment:
Why is this a separate constant, and not (for example) `mem::size_of::<libc::kinfo_file>()`?

Reply:
yes that should work too

Reply:
It seems likely to also be more correct, right?

I don't know that we have any tests in place today for this code path in CI, so is this something you've verified works locally perhaps?

Reply:
In C this constant does exist, it s more a "shortcut" in this case.